r-package/grf/tests/benchmarks/#causal_benchmark.R#

```diff
@@ -69,7 +69,7 @@ make_causal_forest = function(stabilize.splits, min.node.size,
 
 res.untuned.unstab = sapply(1:5, function(setup) {
   evaluate_method(function(X, Y, W) {
-    cf = causal_forest(X, Y, W, tune.parameters = FALSE, stabilize.splits = FALSE)
```

Comment:
No action on your part, but I noticed this file should've never been committed... I guess the `benchmarks` directory could use a clean-up.

Reply:
Yes, there is also a new benchmark dgp collection here: https://github.com/grf-labs/grf/tree/master/experiments/benchmarking